lib/js/components/PopOver/PopOver.stories.ts

```diff
@@ -46,14 +46,59 @@ const argTypes = {
 		control: { type: 'select', options: Object.values(POP_OVER_TRIGGER_ACTIONS) },
 		defaultValue: POP_OVER_TRIGGER_ACTIONS.CLICK,
 	},
+	size: {
+		control: { type: 'select', options: Object.values(POP_OVER_SIZES) },
+		defaultValue: POP_OVER_SIZES.SMALL,
+	}
 } as ArgTypes;
 
-Interactive.argTypes = argTypes;
-Interactive.args = args;
-
 Interactive.parameters = {
 	design: {
 		type: 'figma',
 		url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=3590%3A67741',
 	},
 };
+
+const TooltipStoryTemplate: StoryFn<typeof PopOver> = (argTypes) => ({
+	components: { PopOver },
+	props: Object.keys(argTypes),
+	template:
+		'<div style="display: flex; align-items: center; justify-content: center; height: 600px">' +
+		'<pop-over :placement="placement" :color="color" :trigger-action="triggerAction" :title-text="titleText" :subtitle-text="subtitleText" :button-text="buttonText" :force-show="forceShow" :header-image-url="headerImageUrl" :size="size" :max-height="maxHeight">' +
+		'<template #reference><span>click me!</span></template>' +
+		'<div>Bacon ipsum dolor amet t-bone meatball ground roundBacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork. Bacon ipsum dolor amet t-bone meatball ground round turducken buffalo pork.  </div>' +
+		'</pop-over>' +
+		'</div>',
+});
+
+export const Tooltip = TooltipStoryTemplate.bind({});
+
+Tooltip.args = {
+	placement: POP_OVER_PLACEMENTS.BOTTOM,
+	color: POP_OVER_COLORS.DEFAULT,
+	triggerAction: POP_OVER_TRIGGER_ACTIONS.CLICK,
+	titleText: 'Lorem ipsum',
+	subtitleText: 'Dolor sit amet',
+	size: POP_OVER_SIZES.MEDIUM,
+	forceShow: false,
+	maxHeight: true,
+} as Args;
+
+Tooltip.argTypes = {
```

Comment:
what's the difference between the original story? Maybe we should state that in the story name?